sharktank/sharktank/models/llama/llama.py

```diff
@@ -305,10 +305,6 @@ def forward(
                 xq=xq, xk=xk, mask=embedding_batch_mask
             )
 
-        # TODO: Some model variants do some form of kv repetition to expand the
-        # count of kv heads to the count of attention heads used by the q.
-        assert self.head_count == self.head_count_kv, "NYI: KV expansion"
```

Comment:
Thanks, this also helped me get `mistral-7b-v0.1.Q8_0.gguf` from https://huggingface.co/TheBloke/Mistral-7B-v0.1-GGUF running!

Reply:
Take over/land it off you have a test setup that works.

Reply:
Tested `llama3_8B_q8_0` and got it to complete `python -m sharktank.examples.paged_llm_v1 --hf-dataset=llama3_8B_q8_0 "1, 2, 3, 4"` with output like
```
:: Result tokens: [[11, 220, 20, 11, 220, 21, 11, 220, 22]]
  0(9, 19): , 5, 6, 7
```
Good enough? Not sure how else we want to validate.

Reply:
Well... dunno what to expect, some of these aren't commas :P

```
:: Result tokens: [[11, 220, 20, 11, 220, 21, 11, 220, 22, 11, 220, 23, 0, 220, 24, 11, 220, 605, 11, 220, 806, 11, 220, 717, 11, 220, 1032, 0, 220, 975, 11]]
  0(31, 41): , 5, 6, 7, 8! 9, 10, 11, 12, 13! 14,
```

Reply:
Good enough for now. We need to do a later by layer comparison and test. Did that a month ago and needs someone to go through the details again. Part of the bringup process.